tooling/Microsoft.VisualStudio.Mac.RazorAddin/Microsoft.VisualStudio.Mac.RazorAddin.csproj

```diff
@@ -42,15 +42,6 @@
   </ItemGroup>
 
   <ItemGroup>
-    <!-- 
-      The extension project can not have a direct reference to its language service pieces. They are included via the manifest above.
```

Comment:
Some of our previous expectations about addins were wrong (not being able to directly reference your libraries). To avoid using reflection to bootstrap our types I tried out directly referencing our libraries and all worked fine.

Reply:
Cool 👍